src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -84,6 +89,13 @@ export class EditorComponent implements OnInit {
   scaledCanvas: ElementRef<HTMLCanvasElement>; 
   private scaledCtx: CanvasRenderingContext2D;
 
+  // Overlays custom "cursor" that changes based on brush size.
+  @ViewChild('cursorCanvas', { static: true })
+  cursorCanvas: ElementRef<HTMLCanvasElement>;
+  private cursorCtx: CanvasRenderingContext2D;
+  cursorTarget;
+  maskTarget;
```

Comment:
Are these necessary? You just make them reference the cursor canvas and scaledCanvas right? but the reference already exists in cursorCanvas and scaledCanvas. So I would just use those references and not make a second element ref.

Reply:
I believe you did it the angular way, which didn't work with how I implemented it, typescript doesn't like the typing. Thus, I had to make new vars that essentially referenced the same thing in order to use the event listener methods.